product-pricing-service/src/main/java/com/hedvig/contract/service/ContractService.kt

```diff
@@ -36,6 +37,7 @@ interface ContractService {
     fun activatePendingAgreementOfContract(contractId: UUID, request: ActivatePendingAgreementRequest, token: String?)
     fun terminateContract(contractId: UUID, terminationRequest: TerminateContractRequest, token: String?)
     fun revertContractTermination(contractId: UUID, token: String?)
+    fun terminateAllContractsForMember(memberId: String, terminationRequest: TerminateContractRequest, token: String?): AllContractsTerminationState
```

Comment:
We don't need a token unless it's used from hope
```suggestion
    fun terminateAllContractsForMember(memberId: String, terminationRequest: TerminateContractRequest): AllContractsTerminationState
```